packages/flutter/lib/src/widgets/media_query.dart

```diff
@@ -351,6 +354,31 @@ class MediaQueryData {
   /// gesture behavior over the framework constants.
   final DeviceGestureSettings gestureSettings;
 
+  /// {@macro dart.ui.ViewConfiguration.displayFeatures}
+  ///
+  /// See also:
+  ///
+  ///  * [dart:ui.DisplayFeatureType], where you can see types of display features.
```

Comment:
nit (here and elsewhere): avoid "you", see https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#use-the-passive-voice-recommend-do-not-require

Maybe just say ", which lists the different types of display features."